doc/whats_new/v1.1.rst

```diff
@@ -206,6 +206,9 @@ Changelog
   `-1` and the original warning message is shown.
   :pr:`22217` by :user:`Meekail Zain <micky774>`.
 
+- |Enhancement| :class:`cluster.Birch` now preserves dtype for `numpy.float32`
+  inputs. :pr:`22968` by `Meekail Zain <micky774>`.
+
```

Comment:
1.1.0rc1 was released a few days ago and acts as a feature freeze. This should now be moved to the 1.2 what's new